StudCity.Frontend/src/utils/validators/registerCompleteValidators.js

```diff
@@ -1,4 +1,6 @@
 const PHONE_NUMBER_VALID_LENGTH = 13;
+const ROLE_STUDENT = 1;
```

Comment:
please use strings instead

Reply:
This is good variant because in backend this automatically mapping to ENUM type